dto/src/main/java/org/jboss/pnc/dto/BuildConfiguration.java

```diff
@@ -74,6 +84,6 @@ private BuildConfiguration(SCMRepository repository, ProjectRef project, Environ
     }
 
     @JsonPOJOBuilder(withPrefix = "")
-    public static final class Builder {
+    public static class Builder {
```

Comment:
Why have you removed the final from the Builder?

Reply:
Probably by mistake, adding it back...